### PR TITLE
Revising .jshintrc

### PR DESCRIPTION
**Pull request by gvn**

### CODE REVIEW

File: linters/.jshintrc [hunk 1]
```diff
@@ -1,26 +1,13 @@
 {
-  "globals": {
```
Comment (gvn) [inline linters/.jshintrc]: Adding both `browser` and `node` environments eliminates the need for this list.

File: linters/.jshintrc [hunk 2]
```diff
@@ -1,26 +1,13 @@
...
-    "__dirname": false,
-    "__filename": false,
-    "console": false,
-    "module": false,
-    "process": false,
-    "require": false
-  },
-  "camelcase": true,
```
Comment (gvn) [inline linters/.jshintrc]: Deprecated

File: linters/.jshintrc [hunk 3]
```diff
@@ -1,26 +1,13 @@
...
   "bitwise": true,
   "browser": true,
   "curly": true,
   "eqeqeq": true,
   "freeze": true,
-  "immed": true,
```
Comment (gvn) [inline linters/.jshintrc]: Deprecated

File: linters/.jshintrc [hunk 4]
```diff
@@ -1,26 +1,13 @@
...
-  "indent": 2,
```
Comment (gvn) [inline linters/.jshintrc]: Deprecated

File: linters/.jshintrc [hunk 5]
```diff
@@ -1,26 +1,13 @@
...
   "latedef": true,
-  "newcap": true,
```
Comment (gvn) [inline linters/.jshintrc]: Deprecated

File: linters/.jshintrc [hunk 6]
```diff
@@ -1,26 +1,13 @@
...
-  "noempty": true,
```
Comment (gvn) [inline linters/.jshintrc]: Deprecated

File: linters/.jshintrc [hunk 7]
```diff
@@ -1,26 +1,13 @@
...
-  "quotmark": true,
```
Comment (gvn) [inline linters/.jshintrc]: Deprecated

File: linters/.jshintrc [hunk 8]
```diff
@@ -1,26 +1,13 @@
...
-  "trailing": true,
```
Comment (gvn) [inline linters/.jshintrc]: Deprecated

File: linters/.jshintrc [hunk 9]
```diff
@@ -1,26 +1,13 @@
...
+  "node": true,
+  "notypeof": true,
+  "shadow": true,
   "undef": true,
-  "unused": "vars",
-  "maxlen": 80
```
Comment (gvn) [inline linters/.jshintrc]: Deprecated

**Comment by gvn** [comment]
Note: You have to look at the source to see a comprehensive list of variables that `browser` and `node` whitelist.

https://github.com/jshint/jshint/blob/bfcc86d6e51f53ae1e05cfdd33618e9bc0a9a957/src/vars.js

File: linters/.jshintrc (hunk 9 above)
Comment (thisandagain) [inline linters/.jshintrc]: :cry: LOL ... :wink:

Comment (gvn) [inline linters/.jshintrc]: FWIW: I think we should have a maxlength defined in JSCS. 100 should probably be good for 2 spaces...or we could keep 128 like in WMA.

Comment (k88hudson) [inline linters/.jshintrc]: :+1:

File: linters/.jshintrc [hunk 10]
```diff
@@ -1,26 +1,13 @@
...
   "freeze": true,
-  "immed": true,
-  "indent": 2,
   "latedef": true,
-  "newcap": true,
-  "noempty": true,
-  "quotmark": true,
-  "trailing": true,
+  "node": true,
+  "notypeof": true,
```
Comment (Pomax) [inline linters/.jshintrc]: why disallow typeof checks?

File: linters/.jshintrc (hunk 9 above)
Comment (Pomax) [inline linters/.jshintrc]: whatever number is longer O_O

File: linters/SublimeLinter/SublimeLinter.sublime-settings [hunk 11]
```diff
@@ -13,30 +13,16 @@
...
-      "maxlen": 80
-    }
+    "bitwise": true,
+    "browser": true,
+    "curly": true,
+    "eqeqeq": true,
+    "freeze": true,
+    "latedef": true,
+    "node": true,
+    "notypeof": true,
```
Comment (Pomax) [inline linters/SublimeLinter/SublimeLinter.sublime-settings]: same here, why are we disallowing typeof checks?

File: linters/.jshintrc (hunk 9 above)
Comment (thisandagain) [inline linters/.jshintrc]: I vote 128 as to keep it consistent.

File: linters/.jshintrc (hunk 10 above)
Comment (gvn) [inline linters/.jshintrc]: That's actually not what it does, but we should actually remove this! 

I didn't realize this *suppresses* warnings (not sure why this isn't under relaxing options, but whatever...).

Comment (gvn) [inline linters/.jshintrc]: http://jshint.com/docs/options/#notypeof

**Pull request merged**